tools/bin/acceptance_test_kube_helm.sh

```diff
@@ -77,21 +78,19 @@ echo "Printing docker disk usage..."
 docker system df
 
 if [ -n "$CI" ]; then
-  echo "Pruning all images..."
-  docker image prune --all --force
+ echo "Pruning all images..."
+ docker image prune --all --force
 
-  echo "Printing system disk usage after pruning..."
-  df -h
+ echo "Printing system disk usage after pruning..."
+ df -h
 
-  echo "Printing docker disk usage after pruning..."
-  docker system df
+ echo "Printing docker disk usage after pruning..."
 fi
+ docker system df
```

Comment:
Should it be inside `if` statement above?

Reply:
It's the same as with kube ac tests, idk, but this command doesn't do anything harmful so we can leave it there